### ISSUE TITLE
StripeRawJsonObject returned instead of Source when unmarshalling an Event source.chargeable

### ISSUE THREAD

**Issue by mguillet** [stripe/stripe-java#347]
When creating a new source, our backend receive this kind of event: [event.json](https://github.com/stripe/stripe-java/files/788102/event.txt)

The object associated to this event is supposed to be a [Source](https://stripe.com/docs/api#sources).

That's not what the current API (3.10.0) is returning me:
```java
Event event = APIResource.GSON.fromJson(eventJson, Event.class);
StripeObject object = event.getData().getObject();
```
The `object` I get is a `StripeRawJsonObject` instead of a `Source`. To get a source, I have to do something like this:

```java
JsonObject hack = APIResource.GSON.fromJson(object.toJson(), JsonElement.class).getAsJsonObject();
JsonObject sourceJson = hack.getAsJsonObject("json");
Source source = APIResource.GSON.fromJson(sourceJson, Source.class);
```

Am I missing something obvious or `APIResource.GSON.fromJson` is not doing entirely its job?

**Comment by ob-stripe** [comment]
Hi @mguillet, you're entirely correct: it looks like we forgot to add source objects to the event deserializer. I have prepared a patch here: #348 and we will release an updated version soon.

Thanks for reporting this and sorry for the inconvenience!

**Comment by mguillet** [comment]
Thanks @olivierbellone for the quick feedback! Feel free to close this issue whenever your PR is merged.

**Comment by ob-stripe** [comment]
Closed #347 via #348.
